nemo/collections/llm/tools/auto_configurator/core/training_config.py

```diff
@@ -250,288 +286,287 @@ def init_params(self):
                 self.tp = [4, 8]
                 self.pp = [1, 2, 4]
                 self.mbs = [1, 2, 4]
                 self.min_model_parallel = 4
                 self.max_model_parallel = 32
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 23.0:
                 self.tp = [2, 4, 8]
                 self.pp = [x for x in self.valid_pp if 1 <= x <= 8]
                 self.min_model_parallel = 8
                 self.max_model_parallel = 64
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 45.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 1 <= x <= 12]
                 self.mbs = [1, 2, 4]
                 self.min_model_parallel = 16
                 self.max_model_parallel = 128
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 95:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 1 <= x <= 16]
                 self.mbs = [1, 2, 4]
                 self.min_model_parallel = 16
                 self.max_model_parallel = 256
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 130.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 2 <= x <= 26]
                 self.mbs = [1, 2]
                 self.min_model_parallel = 32
                 self.max_model_parallel = 512
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 195.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 2 <= x <= 32]
                 self.mbs = [1, 2]
                 self.min_model_parallel = 64
                 self.max_model_parallel = 1024
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 395.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 4 <= x <= 64]
                 self.mbs = [1, 2]
                 self.min_model_parallel = 128
                 self.max_model_parallel = 2048
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 790.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 8 <= x <= 128]
                 self.mbs = [1, 2]
                 self.min_model_parallel = 256
                 self.max_model_parallel = 4096
-                self.gbs = 2048
+                self.gbs = [2048]
             elif model_size_in_b <= 1100.0:
                 self.tp = [4, 8]
                 self.pp = [x for x in self.valid_pp if 8 <= x <= 192]
                 self.mbs = [1, 2]
                 self.min_model_parallel = 512
                 self.max_model_parallel = 8192
-                self.gbs = 2048
+                self.gbs = [2048]
```

Comment:
Again, I know you didn't write this, but this feels like it is way too specific and not modular enough for different model types/sizes, and GPU types. For example, what happens if a user wants `seq_length` of 128k? Or they use H200s which have 144 GB of memory? Does your code touch this declaration at all?